### DISCUSSION TITLE
2023-07-02-hongkong

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#144] [General]
# 香港和想象中不一样的地方 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share第一次去香港，平时旅游不太喜欢赶场打卡，反而喜欢观察人文环境，发现不少和想象中不一样的地方，放到这里随便聊聊。

https://tw93.fun/2023-07-02/hongkong.html

**Comment by snakeliwei** [comment]
有些图片挂了

**Reply by tw93** [reply]
感谢反馈，我已经修复了

**Comment by hoganzhao** [comment]
这个字体真心好看，是什么字体

**Reply by tw93** [reply]
了解一下 https://github.com/lxgw/LxgwWenKai
此外妙言的默认字体也是用的这个

**Comment by mayuyuyu** [comment]
文字让我想起了自己第一次去香港时观察周遭的感觉，后面去多了也就像待在常驻城市总是奔赴一个个目的地，忽视沿途的人文。

**Reply by tw93** [reply]
是的，因为第一次，所以会好奇心很重

**Comment by lee2333** [comment]
前天刚从港澳回来，恰好时隔多年再去，相比于高速发展的大陆感觉那边已经没有第一次去的时候国际大都市的感觉了，本打算以后旅行不会考虑香港了。但是昨天恰好去家门口的K11母婴室发现没有热水，甚至四层商场只有一个母婴室，对比之下香港的无障碍和母婴照顾做的确实还是值得学习的。

**Comment by Beeta** [comment]
「菲佣姐姐周日出来，是因为僱主不让她们在家里，僱主周日在家需要隐私。」
这个不一定对，很多雇主都希望菲佣一直照看孩子的，但菲佣坚持要有周末。

**Reply by anguschiu1** [reply]
菲傭的工作場所就是僱主的家，週日是法定假期，他不用工作，但又多不會熟絡到會坐在沙發上和僱主一同看電視的程度，所以外出和同鄉玩就是他們最輕鬆的環境了。

如果OP週日週一再去同一地方如中環，你會發現菲傭姐姐聚會點全變成白領人士趕路的地方，甚至懷疑自己是不是搭錯車去了另一個地方，好有趣的。

**Comment by lonelili** [comment]
下次有机会可以去香港麦理浩径走一走。香港大部分人的印象是大都市，其实它的自然资源非常丰富，保护得也非常好，这一点非常值得称赞，

**Reply by anguschiu1** [reply]
麥徑很美麗，1，2，8，9，10段比較易行，但3，4，5，6，7並不輕鬆，有很多山嶺要跨，包括香港最高的大帽山，出發前記得留意，也最好不要單獨行動呢。

**Reply by tw93** [reply]
牛牛牛！
